spec/serializers/search_and_compare/course_serializer_spec.rb

```diff
@@ -237,9 +237,8 @@
                 site_status.site.address1,
                 site_status.site.address2,
                 site_status.site.address3,
-                site_status.site.address4,
-                site_status.site.postcode
-              ].reject(&:blank?).join('\n')
+                site_status.site.address4
+              ].reject(&:blank?).join('\n') + (site_status.site.postcode.present? ? ' ' + site_status.site.postcode : '')
```

Comment:
Is this right? Doesn't the change above join with a comma (`,`)? So why is this test passing looking for newlines?

Reply:
fixed, good catch at the time I did noticed test was passing without any changes, as the one and only json we pluck out has an invalid site saved.